lib/protobuf/dsl/typespecs.ex

```diff
@@ -36,14 +36,16 @@ defmodule Protobuf.DSL.Typespecs do
         _other -> []
       end
 
-    unknown_varints_fields =
-      if field_name = message_props.unknown_varints_field do
-        [{field_name, quote(do: [{field_number :: integer(), value :: integer()}])}]
-      else
-        []
-      end
-
-    field_specs = regular_fields ++ oneof_fields ++ extension_fields ++ unknown_varints_fields
+    unknown_fields = [
+      {:__unknown_fields__,
+       quote(
+         do: [
+           {field_number :: integer(), Protobuf.Wire.Types.wire_type(), value :: integer()}
```

Comment:
I guess the value can be anything now?

```suggestion
           {field_number :: integer(), Protobuf.Wire.Types.wire_type(), value :: term()}
```